CMakeLists.txt

```diff
@@ -281,7 +281,7 @@ if (NOT EMSCRIPTEN)
   endif ()
   set_property(
     DIRECTORY APPEND PROPERTY ADDITIONAL_MAKE_CLEAN_FILES
-    "wasm2wast" "wast2wasm")
+    "wasm2wast" "wast2wasm" "sexpr-wasm" "wasm-wast")
```

Comment:
Shouldn't you remove the wasm2wast and wast2wasm? I assumed those would already be cleaned.